src/Catalog/PackageCatalogItem.cs

```diff
@@ -103,23 +107,23 @@ public override IGraph CreateContentGraph(CatalogContext context)
             //  identity and version
             SetIdVersionFromGraph(graph);
 
-            // deprecation
+            //  deprecation
             if (Deprecation != null)
             {
-                // assert deprecation root node to subject
+                //  assert deprecation root node to subject
```

Comment:
nit: double space after `//` is nonstandard

Reply:
I was making the comments in the file consistent (see others that pre-date this change), but I can change them all to 1 space, sure.